src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -14,9 +14,24 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     ArrayList<MentorObject> mentors = new ArrayList<MentorObject>();
-    mentors.add(new MentorObject("Obi", "Obi is awesome."));
-    mentors.add(new MentorObject("Sami", "Sami is fun."));
-    mentors.add(new MentorObject("Richi", "Richi is cool."));
+    ArrayList<String> projects = new ArrayList<String>();
+    projects.add("OpenSesame");
+
+    ArrayList<String> ObiSkills = new ArrayList<String>();
+    ObiSkills.add("Meme god");
+    ObiSkills.add("HTML wrangler");
+    mentors.add(new MentorObject("Obi", "Obinnabii", "Obi is awesome.", ObiSkills, projects));
+
+    ArrayList<String> SamiSkills = new ArrayList<String>();
+    SamiSkills.add("Stone carver");
+    SamiSkills.add("Bootstrap convert");
+    mentors.add(new MentorObject("Sami", "Sami-2000", "Sami is fun.", SamiSkills, projects));
+
+    ArrayList<String> RichiSkills = new ArrayList<String>();
+    RichiSkills.add("Minecraft boss");
+    RichiSkills.add("React wizard");
+    mentors.add(new MentorObject("Richi", "Richie78321", "Richi is cool.", RichiSkills, projects));
+
     String jsonMentors = new Gson().toJson(mentors);
```

Comment:
nit: Feel free to instantiate this Gson object only once. From the [user guide](https://github.com/google/gson/blob/master/UserGuide.md#TOC-Gson-Performance-and-Scalability):
> The Gson instance does not maintain any state while invoking Json operations. So, you are free to reuse the same object for multiple Json serialization and deserialization operations.

Reply:
Quick clarifying Q: Where else do I instantiate a Gson object? (I only see a Gson object declaration on line 36)

Reply:
You can instantiate it once when the class `MentorServlet` is instantiated like this (not perfect but you get the point): 
```java
class MentorServlet {
  private Gson gson = new Gson();
  [...]
}
That way you are not recreating the Gson object every time a get request is made.